docs/eks.md

```diff
@@ -0,0 +1,170 @@
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        customresourcedefinition.apiextensions.k8s.io/perconapgbackups.pg.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/perconapgclusters.pg.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/perconapgrestores.pg.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/postgresclusters.postgres-operator.crunchydata.com serverside-applied
+        serviceaccount/percona-postgresql-operator serverside-applied
+        role.rbac.authorization.k8s.io/percona-postgresql-operator serverside-applied
+        rolebinding.rbac.authorization.k8s.io/service-account-percona-postgresql-operator serverside-applied
+        deployment.apps/percona-postgresql-operator serverside-applied
+        ```
+
+    As the result you will have the Operator Pod up and running.
+
+2. The operator has been started, and you can deploy your Percona Distribution
+    for PostgreSQL cluster:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/cr.yaml -n postgres-operator
+    ```
+
+    ??? example "Expected output"
+
+        ``` {.text .no-copy}
+        erconapgcluster.pg.percona.com/cluster1 created
+        ```
+
+    !!! note
+
+        This deploys default Percona Distribution for PostgreSQL configuration.
+        Please see [deploy/cr.yaml](https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/cr.yaml)
+        and [Custom Resource Options](operator.md) for the configuration
+        options. You can clone the repository with all manifests and source code
+        by executing the following command:
+
+        ```{.bash data-prompt="$" }
+        $ git clone -b v{{ release }} https://github.com/percona/percona-postgresql-operator
+        ```
+
+        After editing the needed options, apply your modified `deploy/cr.yaml`
+        file as follows:
+
+        ```{.bash data-prompt="$" }
+        $ kubectl apply -f deploy/cr.yaml -n postgres-operator
+        ```
+
+    The creation process may take some time. The process is over when both
+    Operator and replica set Pods have reached their Running status:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl get pg
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        NAME       ENDPOINT                     STATUS   POSTGRES   PGBOUNCER   AGE
+        cluster1   cluster1-pgbouncer.pgo.svc   ready    3          3           143m
+        ```
+
+
+## Verifying the cluster operation
+
+When creation process is over, `kubectl get pg` command will show you the
+cluster status as `ready`, and you can try to connect to the cluster.
+
+{% include 'assets/fragments/connectivity.txt' %}
+
+## Removing the EKS cluster
```

Comment:
I don't think we need to have it. We do not have commands on how to create eks cluster. Why do we need to have the command on how to remove it?